custom_components/tuya_local/devices/arlec_apr005ha_air_purifier.yaml

```diff
@@ -0,0 +1,101 @@
+name: Air Purifier
+products:
+  - id: dfgsetrdnabqeimr
+    name: Arlec Air Purifier APR005HA
+primary_entity:
+  entity: fan
+  translation_only_key: fan_with_presets
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 3
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: Auto
+          value: smart
+        - dps_val: Manual
+          value: normal
+        - dps_val: Sleep
+          value: sleep
+    - id: 4
+      type: integer
+      name: speed
+      range:
+        min: 1
+        max: 3
+secondary_entities:
+  - entity: switch
+    translation_key: ionizer
+    category: config
+    dps:
+      - id: 6
+        type: boolean
+        name: switch
+  - entity: switch
+    translation_key: uv_sterilization
+    category: config
+    dps:
+      - id: 9
+        type: boolean
+        name: switch
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 18
+        type: string
+        name: option
+        mapping:
+          - dps_val: cancel
+            value: "Off"
+          - dps_val: 1h
+            value: 1 hour
+          - dps_val: 2h
+            value: 2 hours
+          - dps_val: 3h
+            value: 3 hours
+          - dps_val: 4h
+            value: 4 hours
+          - dps_val: 5h
+            value: 5 hours
+          - dps_val: 6h
+            value: 6 hours
+          - dps_val: 7h
+            value: 7 hours
+          - dps_val: 8h
+            value: 8 hours
+          - dps_val: 9h
+            value: 9 hours
+          - dps_val: 10h
+            value: 10 hours
+          - dps_val: 11h
+            value: 11 hours
+          - dps_val: 12h
+            value: 12 hours
+  - entity: sensor
+    name: Air quality
```

Comment:
It may be better to leave the name out so that translations can be used.

Reply:
Sounds good, cheers

Reply:
ah that's right, without the names everything becomes air quality, should I put the name back in or is there a better way to do it?
![image](https://github.com/user-attachments/assets/377e4b92-9c94-491e-9967-c3b4480d7256)

Reply:
I think that was a side effect of an earlier fix that is now reverted.